bitcoin-review-podcast/cashu-and-fedimint:-ecash-bitcoin's-layer-3-and-scalability.md

```diff
@@ -0,0 +1,2499 @@
+Well, I might as well give it.
+It's because Bitcoin didn't exist.
+So you didn't have a source of truth.
+You didn't have a way to peg in and peg out.
+You didn't have a way of interacting as an on and off ramp with a system that's digital without digital money, right?
+So like match made in heaven, This thing is now actually usable.
+It's super cool.
+Very well proven technology, very well studied.
+But then comes Eric and Calle and like take this thing to a whole other level And because of Bitcoin, because of Lightning.
+And I hope we can sort of explore this in depth here.
+So who's going to volunteer to give a little bit of a technical primer of what is eCash, what is eCash now, and sort of like where we're at on how you guys are exploring it.
+
+Calle: 00:04:26
+
+So I'm going to try it here.
+First of all, maybe just a comment on what you said.
+I'm not so sure that DigiCash, which is the company by David Chaum, who invented this whole thing in 1982, DigiCash, the company was founded eight years later.
+So 89 or 90 or something.
+So there was a pretty long gap between those two events.
+And DigiCash itself tried to be viable for almost a decade, but I think active it was for maximum five years or so.
+And when you look back on the reasons why it failed, It's not usually because the fiat system was so prohibitive.
+So that's also my take.
+I share the take with you that e-cash needs Bitcoin to really shine.
+But when you look back, the reasons why DigiCash itself failed was literally poor management.
+So there were insanely huge deals on the table.
+There was Deutsche Bank was interested, but that wasn't even the biggest one, I think.
+Microsoft- 
+
+Rijndael: 00:05:26
+
+Do you know Microsoft want to put it in Windows or something?
+
+Calle: 00:05:28
+
+It's insane.
+They wanted to put a DigiCash wallet into every Windows 95 installation, but David Chaum then wanted too big of a share of the income there, or the money generated, so Microsoft pulled out.
+
+NVK: 00:05:43
+
+He has a history of expecting a little too much and wanting too much of a set of things.
+
+Calle: 00:05:47
+
+And the third one is Visa.
+Visa offered them 40 million back in the 90s.
+That was still real money, right?
+And he just said, no, I want double of it just in the last couple of days or so.
+So the deal also failed.
+So my way of seeing, looking at this is DigiCash then crashed or was, you know, just a closed shop.
+And then a couple of years later, PayPal and credit cards on the internet really blew up.
+So I think...
+
+NVK: 00:06:16
+
+How convenient.
+
+Calle: 00:06:17
+
+Yeah, I think so.
+If things would have gone a little bit more different back in the 90s, I think the default online payment experience today could have been, without Bitcoin and everything before, could have been perfectly private e-cash on the fiat system.
+That is technically possible.
+I know regulatory problems then started with, you know, everything that's happened later.
```

Comment:
Observing the transcript, it's clear that the phrase ', you know,' is frequently employed as a filler, with 93 occurrences. To improve overall clarity and coherence, consider removing it where possible.